unified_planning/solvers/__init__.py

```diff
@@ -27,6 +27,6 @@
             'Parallel',
             'PDDLSolver',
             'SequentialPlanValidator',
-            'Solver', 'OptimalityGuarantee',
+            'Solver', 'OptimalityGuarantee', 'Credits',
```

Comment:
At this point, we could move `Credits` in `environment.py` or in a module at top-level, no?